Demo/EarlGreyExample/README.md

```diff
@@ -11,11 +11,10 @@ different functionalities provide by EarlGrey, written in Objective-C and Swift.
 Getting Started
 ---------------
 
-Execute `pod install` from the directory where you have downloaded the source of the example, then
-`open EarlGreyExample.xcworkspace` to open the example in xcode. Ensure that your version of
-CocoaPods is upgraded to the latest 1.x version since this project uses the [1.0.0 syntax]
-(http://blog.cocoapods.org/CocoaPods-1.0/). You can get the latest CocoaPods pre-release version by
-running `gem install cocoapods --pre`.
-
+Execute `gem install earlgrey` then `pod install` from the directory where you have downloaded
```

Comment:
I was talking about removing the configure_earlgrey_pods.rb script here. Yeah we could discuss this further on Slack.